massa-client/src/main.rs

```diff
@@ -49,17 +51,12 @@ struct Args {
     #[structopt(name = "PARAMETERS")]
     parameters: Vec<String>,
     /// Path of wallet folder
-    #[structopt(
-        short = "w",
-        long = "wallet",
-        parse(from_os_str),
-        default_value = "wallets/"
-    )]
+    #[structopt(short = 'w', long = "wallet", default_value = "wallets/")]
```

Comment:
Why is there still references to structopt? Did clap integrate these macros?

Reply:
Removed :)